### PR TITLE
[BUGFIX] Correct handling of rounding directive in NumericMetricRangeMultiBatchParameterBuilder

### PR DESCRIPTION
**Pull request by alexsherstinsky**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
- JIRA: https://superconductive.atlassian.net/browse/GREAT-804
-
-

After submitting your PR, CI checks will run and @ge-cla-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added [unit tests](https://docs.greatexpectations.io/docs/contributing/contributing_test#writing-unit-and-integration-tests) where applicable and made sure that new and existing tests are passing.
- [x] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: great_expectations/rule_based_profiler/data_assistant/volume_data_assistant.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 )
 
 
+# TODO: <Alex>ALEX Rename this "VolumeDataAssistant" to be more precise.</Alex>
```
Comment (cdkini) [inline great_expectations/rule_based_profiler/data_assistant/volume_data_assistant.py:22]: What's the timeline for this change? A subsequent PR?

File: great_expectations/rule_based_profiler/data_assistant/data_assistant.py [hunk 2]
```diff
@@ -163,7 +163,7 @@ def run(
         expectation_suite_name: Optional[str] = None,
         include_citation: bool = True,
     ) -> DataAssistantResult:
-        data_assistant_cls: ABCMeta = type(self)
+        data_assistant_cls: type = type(self)
```
Comment (NathanFarmer) [inline great_expectations/rule_based_profiler/data_assistant/data_assistant.py:166]: 👍🏻

File: tests/conftest.py [hunk 3]
```diff
@@ -3590,7 +3590,7 @@ def bobby_columnar_table_multi_batch(empty_data_context):
                 "kwargs": {
                     "column": "fare_amount",
                     "min_value": 228.94,
-                    "max_value": 2990.05,
+                    "max_value": 2990.06,
```
Comment (cdkini) [inline tests/conftest.py:3593]: Why the change here?

**Review by cdkini — APPROVED** [review]
LGTM!

File: great_expectations/rule_based_profiler/data_assistant/volume_data_assistant.py (hunk 1 above)
Comment (alexsherstinsky) [reply great_expectations/rule_based_profiler/data_assistant/volume_data_assistant.py:22]: @cdkini Waiting for a JIRA containing the "final" name.  Might take a week. /cc @talagluck @NathanFarmer

File: tests/conftest.py (hunk 3 above)
Comment (alexsherstinsky) [reply tests/conftest.py:3593]: @cdkini This is the key part of this PR -- rounding was handled incorrectly and also the builder was run twice instead of once.

**Review by NathanFarmer — APPROVED** [review]
LGTM
